app/assets/stylesheets/yahoo_finance.scss

```diff
@@ -0,0 +1,64 @@
+// Place all the styles related to the yahoo_finance controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+body.sent {
+  background-image: none;
+}
+
+.symbol {
+  font-size: 16px;
+}
+
+.centered {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.tt-input,
+.tt-hint {
+  width: 396px;
+  height: 34px;
+  padding: 8px 12px;
+  font-size: 16px;
+  line-height: 30px;
+  border: 2px solid #ccc;
+  border-radius: 2px;
+  outline: none;
+  vertical-align: middle !important;
+}
+
+.tt-input {
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+}
+
+.tt-hint {
+  color: #999;
+}
+
+.tt-menu {
+  width: 396px;
+  padding: 8px 0;
+  background-color: #fff;
```

Comment:
The front end guy in me says best practices are to have a _colors.scss file for every hex code and rgb value but we can do that separately.